libs/core/src/lib/shared/services/value-state-message.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+/**
+ *
+ * This service holds default
+ * value state aria messages
+ * that are needed for accessibility purposes.
+ *
+ * It's intendant to be used with form controls
+ * which supports different value states
+ *
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class ValueStateAriaMessageService {
+    /** Value state "success" message */
+    readonly success = 'Value State Success';
+
+    /** Value state "information" message */
+    readonly information = 'Value State Information';
+
+    /** Value state "warning" message */
+    readonly warning = 'Value State Warning';
+
+    /** Value state "error" message */
+    readonly error = 'Value State Error';
```

Comment:
How are we handling i18n?

Reply:
This can be overridden on app level with translated strings

Reply:
This goes against our current paradigm of allowing the devs to provide translations via inputs, this could be confusing

Reply:
@mikerodonnell89 there are still inputs for these fields which take precedence over these service values.
This service gives us default values for this kind of inputs.
So if app developer wants to redefine default values for components that rely on this it is needed only to override this service and not inputs for every occurrence in the template

Reply:
we have the same approach for core calendar labels